chicken_app/src/Pages/Profile/ConfirmEmail.js

```diff
@@ -1,9 +1,57 @@
-import React from 'react'
+import { React, useState, useRef } from "react";
+import MailIcon from "@mui/icons-material/Mail";
+import Alert from "@mui/material/Alert";
+import AlertTitle from "@mui/material/AlertTitle";
+import { getAuth, sendEmailVerification } from "firebase/auth";
+
+import { Box, ThemeProvider, createTheme, Typography } from "@mui/material";
+import { borders } from "@mui/system";
+import { Button } from "@mui/material";
+
+const theme = createTheme({
```

Comment:
good work learning about themes! if necessary these can be moved to theme.js, see other comments about theme